### PR TITLE
Renaming sample_infections to compute_future_infections

### PR DESCRIPTION
**Pull request by gvegayon**
Renames functions to avoid suggesting stochasticity.

### CODE REVIEW

**Review by dylanhmorris — COMMENTED** [review]
I know the main issue suggested `compute_future_infections` but on review I might suggest

`compute_infections_from_rt`
and
`compute_infections_from_rt_with_feedback`

**Comment by gvegayon** [comment]
> I know the main issue suggested `compute_future_infections` but on review I might suggest
> 
> `compute_infections_from_rt` and `compute_infections_from_rt_with_feedback`
> 
> Thoughts @gvegayon @damonbayer ?

I like that better. Updating

File: model/src/pyrenew/latent/infections.py [hunk 1]
```diff
@@ -107,7 +107,7 @@ def sample(
         n_lead = gen_int_rev.size - 1
         I0_vec = jnp.hstack([jnp.zeros(n_lead), I0])
 
-        all_infections = inf.sample_infections_rt(
+        all_infections = inf.compute_future_infections_rt(
```
Comment (dylanhmorris) [inline model/src/pyrenew/latent/infections.py:110]:
```suggestion
        all_infections = inf. compute_infections_from_rt(
```

**Review by dylanhmorris — COMMENTED** [review]
Missing an update to the function name

File: model/src/pyrenew/latent/infections.py (hunk 1 above)
Comment (gvegayon) [reply model/src/pyrenew/latent/infections.py]: Addressed in 72fec94

**Review by dylanhmorris — APPROVED** [review]
LGTM

**Pull request merged**